make/mt-os/mt-cc3200.mk

```diff
@@ -5,41 +5,59 @@
 
 include make/mt-os/mt-os-common.mk
 
-###
-## COMPILER NAME
-###
-COMPILER ?= arm_15.12.3.LTS
+ifdef XI_TRAVIS_BUILD
+	# For Xively C Client Continuous Integration only.
+	COMPILER ?= arm_15.12.3.LTS
```

Comment:
Yes. I tried to do it quickly but I didn't have a linux machine to run the binary installation process, so this was my short term step.